packages/drupal-yaml/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@skilld/drupal-yaml",
+  "version": "1.0.0",
+  "description": "Drupal YAML reader/writer",
```

Comment:
If you're using Drupal context, please specify "Drupal theme libraries YAML" (libraries.yml)

Reply:
+